populate/src/main/java/bio/terra/pearl/populate/service/StudyPopulator.java

```diff
@@ -174,7 +174,7 @@ protected Study createOrUpdate(Study existingStudy, StudyPopDto popDto, PortalPo
             StudyEnvironment destEnv = portalDiffService.
                     loadStudyEnvForProcessing(existingStudy.getShortcode(), EnvironmentName.sandbox);
             PortalEnvironment destPortalEnv = portalEnvironmentService
-                    .findOne(context.getPortalShortcode(), context.getEnvironmentName()).get();
+                    .findOne(context.getPortalShortcode(), EnvironmentName.sandbox).get();
```

Comment:
Did you mean to commit this change? Are we losing any functionality by hardwiring this to `sandbox` instead of getting it from the passed-in `context`?

Reply:
Yes, this is deliberate.  See the comment on line 170 -- if we're not in overwrite mode, we only ever want to change the sandbox.  Updates to to irb/live environments must be done manually through the UX